assets/pages/regular-contributions/regularContributions.jsx

```diff
@@ -24,7 +24,7 @@ import FormFields from './components/formFields';
 
 // ----- Page Startup ----- //
 
-const contributionType = parseContrib(getQueryParameter('contribType'), 'MONTHLY');
+const contributionType = parseRegularContributionType(getQueryParameter('contribType') || 'MONTHLY');
```

Comment:
I think the 'MONTHLY' is not necessary if you pass it `parseRegularContributionType` will return `MONTHLY` as its default case.

Reply:
That's to deal with the maybe return from `getQueryParameter`, however it does mean I don't need `parseRegularContributionType` to take a maybe 👍